gensim/models/fasttext.py

```diff
@@ -1401,6 +1401,7 @@ def adjust_vectors(self):
 
         """
         if self.bucket == 0:
+            self.vectors = self.vectors_vocab  # no ngrams influence
```

Comment:
Is our serialization smart enough to realize these attributes reference the same object, and avoid serializing both (into two separate files)?

Reply:
It isn't (except in rare case array is so small it goes through normal single=-file pickling), but next-up PR #2892 simply explicitly avoids saving `.vectors` at all, from `FastTextKeyedVectors`, since even where it's different from `.vectors_vocab`, it's still just a  re-calculable cache of full-word-plus-ngram vectors derived from `.vectors_vocab` & `.vectors_ngram`.